src/components/footer/Footer.component.tsx

```diff
@@ -0,0 +1,31 @@
+import fb from '@/assets/icons/socialMedia/fb.svg';
+import instagram from '@/assets/icons/socialMedia/insta.svg';
+import linkedin from '@/assets/icons/socialMedia/linkedin.svg';
+import FooterIcon from '@/components/footerIcon/FooterIcon.component.tsx';
+import { MY_NAME, URL_FB, URL_INSTAGRAM, URL_LINK_IN, URL_PROJECT } from '@/config/config.ts';
+
+import styles from './footer.module.css';
+
+const Footer = () => (
+    <footer className={styles.footer}>
+        <div className={styles.horizontalLine}></div>
+        <div className={styles.content}>
+            <FooterIcon
+                className={styles.social_items}
+                media={[
+                    { icon: fb, url: URL_FB },
+                    { icon: instagram, url: URL_INSTAGRAM },
+                    { icon: linkedin, url: URL_LINK_IN },
+                ]}
```

Comment:
you can create save array to variable and use instead of write content here

Reply:
fixed